ui/src/components/retreat/home/PraiseWallContainer.js

```diff
@@ -179,48 +179,30 @@ const PraiseWallContainer = ({ userObj }) => {
   const MobileView = () => {
     return (
       <Center display={{ base: 'flex', lg: 'none' }}>
-        <HStack align="start" spacing="5">
-          {SplitToChunks(
-            praiseList.filter(
-              (partitionedList) => partitionedList.category == eventCategory
-            ),
-            2
-          ).map((filteredPraise, i) => (
-            <VStack key={i} align="start" w="40vw">
-              {filteredPraise.map((praise, index) => (
-                <PraiseCard
-                  message={praise.message}
-                  key={i * 100 + index}
-                  cardColor={cardBgColors[(i % 3) + (index % 3)]}
-                />
-              ))}
-            </VStack>
+        <PinterestGrid columns={2} columnWidth={150}>
```

Comment:
`columnWidth={135}`? 

it's overflowing in iphoneSE size
<img width="533" alt="Screenshot 2022-02-25 at 15 26 57" src="https://user-images.githubusercontent.com/33679483/155673753-bffb8d17-ed9a-4126-b074-76bf852cf07b.png">